### ISSUE TITLE
Unclear about usage

### ISSUE THREAD

**Issue by leggsimon** [kr1sp1n/node-vault#97]
_tldr; How can I use github auth and call the other vault commands?_

Hi, this looks like a great module but I'm super unclear as to how to use it.

Currently I'm trying to replicate the following using `node-vault`
```
vault auth --method github
vault list secret
```

This is currently what I have:
```js
const vault = require("node-vault")({
	endpoint: process.env.VAULT_ADDR,
	token: process.env.VAULT_AUTH_GITHUB_TOKEN
})

vault.githubLogin({
	token: process.env.VAULT_AUTH_GITHUB_TOKEN
})
	.then(res => {
		console.log(res)
		return vault.list('secret')
	})
	.then(res => {
		console.log(res)
	})
	.catch(error => {
		console.log(error)
	})
```

The issue I'm having here is that the `.githubLogin(...)` resolves but when I call `vault.list('secret')` I get a 403 promise rejection. I dug around the code and it seems you can pass an object as the second parameter to `list` with a `token` value, but I've tried using my `process.env.VAULT_AUTH_GITHUB_TOKEN`, `res.auth.client_token`, res.auth.accessor` & `res.auth.entity_id` but it always rejects with a 403.

**Comment by kr1sp1n** [comment]
No feedback given by @leggsimon so I assume that #93 fixed the issue.
